src/Compilers/CSharp/Portable/Symbols/Symbol.cs

```diff
@@ -1104,23 +1103,11 @@ public ImmutableArray<SymbolDisplayPart> ToDisplayParts(SymbolDisplayFormat form
             CSharpSyntaxNode block,
             CSharpSyntaxNode expression,
             CSharpSyntaxNode syntax,
-            DiagnosticBag diagnostics)
+            DiagnosticBag diagnostics,
+            ErrorCode code)
```

Comment:
All but one caller passes `ErrorCode.ERR_BlockBodyAndExpressionBody`. Consider making that the default value.